localization/english/wc_buildings_l_english.yml

```diff
@@ -1,7 +1,7 @@
 ﻿l_english:
  building_type_dark_portal_01:0 "Dark Portal"
- building_type_dark_portal_01_desc:1 "A gateway that binds the two worlds: Azeroth and Draenor, allowing alien creatures to go back and forth."
+ building_type_dark_portal_01_desc:1 "A gateway that binds the two worlds of Azeroth and Draenor, allowing alien creatures to go back and forth."
  building_dark_portal_01:1 "Dark Rift"
- building_dark_portal_01_desc:1 "A tiny rift in the fabric of the universe that still connects the two worlds."
+ building_dark_portal_01_desc:1 "A tiny rift in the fabric of the universe that still connects the two worlds of Azeroth and Draenor."
  building_dark_portal_02:1 "Dark Portal"
- building_dark_portal_02_desc:1 "A massive arch with magical runes that maintains the connection between the two worlds, allowing large armies to pass through the portal."
+ building_dark_portal_02_desc:1 "A massive arch with magical runes that maintains the connection between the two worlds of Azeroth and Draenor, allowing large armies to pass through the portal."
```

Comment:
The phrase "the two world of Azeroth and Draenor" is used too often in this file, would recommend just using "between Azeroth and Draenor" or "between the worlds of Azeroth and Draenor" in at least one the these lines

Reply:
Ok, can do that. I just thought that adding the names of the worlds would be better than just "the two worlds".